companies-manifest.js

```diff
@@ -870,6 +870,23 @@ const companies = [
       },
     },
   },
+  {
+    key: 'https://www.riotgames.com/en',
+    similarWebGlobalRank: 1819,
+    name: 'Riot Games',
+    logo: null,
```

Comment:
It would be extremely useful to also have the logo for Riot games here as it would be nice to have them on the homepage too. Can you provide their SVG logo?

Reply:
I'll try to find it